src/consensus/default_constants.py

```diff
@@ -53,9 +53,10 @@
     # 1 vByte = 108 clvm cost units
     "CLVM_COST_RATIO_CONSTANT": 108,
     # Max block cost in clvm cost units (MAX_BLOCK_COST * CLVM_COST_RATIO_CONSTANT)
-    "MAX_BLOCK_COST_CLVM": 23077872,
+    # "MAX_BLOCK_COST_CLVM": 23077872,
+    "MAX_BLOCK_COST_CLVM": 40000000,  # Based on arvid analysis
```

Comment:
we don't use this yet, which I believe is fine until we have transactions

Reply:
We do use it after transaction freeze (in testnet)